tests/foreman/cli/test_host.py

```diff
@@ -2046,27 +2055,22 @@ def test_negative_without_attach_with_lce(
         },
         force_use_cdn=True,
     )
-    host_lce = target_sat.api.LifecycleEnvironment(organization=function_org).create()
-    # refresh content view data
-    content_view.publish()
-    content_view.read().version[-1].promote(data={'environment_ids': host_lce.id, 'force': False})
 
     # register client
     host_subscription_client.register_contenthost(
         function_org.name,
-        lce=f'{host_lce.name}/{content_view.name}',
+        lce=f'{function_lce.name}/{content_view.name}',
         auto_attach=False,
     )
 
-    # get list of available subscriptions which are matched with default subscription
-    subscriptions = host_subscription_client.run(
-        f'subscription-manager list --available --matches "{DEFAULT_SUBSCRIPTION_NAME}" --pool-only'
-    )
-    pool_id = subscriptions.stdout.strip()
-    # attach to plain RHEL subsctiption
-    host_subscription_client.subscription_manager_attach_pool([pool_id])
     assert host_subscription_client.subscribed
-    host_subscription_client.enable_repo(REPOS['rhst7']['id'])
+    res = host_subscription_client.enable_repo(REPOS['rhst7']['id'])
+    if sca_enabled:
+        assert res.status == 0
+        assert f"Repository '{REPOS['rhst7']['id']}' is enabled for this system."
+    else:
+        assert res.status != 0
```

Comment:
Isnt it good to verify sca_disable and repository enablement. Or are we strictly going away from doing sca disable and repo enablement ?

Reply:
I'm not saying we shouldn't test it but it's not the purpose of this particular test.

Reply:
Sure! But then:

1. Do we have dedicate test to test this combination, I am sure we have it!
2. If the purpose it not to test disable SCA, could we remove the parametrization for it, to save run time ?

Reply:
Purpose of this PR is to fulfill a request to `Improve the use case for SCA based mode`.